packages/web-components/@sfx/product/src/variant.ts

```diff
@@ -43,17 +42,20 @@ export default class Variant extends Base {
 
   render(): TemplateResult {
     const { variant } = this;
-
-    return html`
-      ${this.type === 'image'
-    ? html`<img
+    const img = this.type === 'image'
+      ? html`<img
           class="sfx-variant-image"
           src="${variant.image}"
           alt="${variant.text}"
         />`
-    : ''
-}
+      : '';
+
+    return html`${img}
       ${this.type === 'text' ? variant.text : ''}
```

Comment:
Is it worth formatting this the same way as `img` for consistency? If text was a little more complex I would say yes for sure, but as it is, i'm not too concerned either way

Reply:
If we want to be really clean, yes, but I'm not too concerned about it as long as it doesn't span multiple lines and we don't need to reuse it.